db/seeds/post.rb

```diff
@@ -0,0 +1,622 @@
+  { user_id: 3, post_id: 20, can_owe: true, can_reply: true, joined: false },
+  { user_id: 2, post_id: 21, can_owe: true, can_reply: true, joined: true, joined_at: "2012-09-30 21:27:00" },
+  { user_id: 3, post_id: 21, can_owe: true, can_reply: true, joined: false },
+  { user_id: 2, post_id: 22, can_owe: true, can_reply: true, joined: true, joined_at: "2012-10-01 17:07:00" },
+  { user_id: 3, post_id: 22, can_owe: true, can_reply: true, joined: false },
+  { user_id: 2, post_id: 23, can_owe: true, can_reply: true, joined: true, joined_at: "2012-10-01 20:08:00" },
+  { user_id: 3, post_id: 23, can_owe: true, can_reply: true, joined: false },
+  { user_id: 2, post_id: 24, can_owe: true, can_reply: true, joined: true, joined_at: "2012-10-01 23:09:00" },
+  { user_id: 3, post_id: 24, can_owe: true, can_reply: true, joined: false },
+  { user_id: 2, post_id: 25, can_owe: true, can_reply: true, joined: true, joined_at: "2012-10-03 19:37:00" },
+  { user_id: 3, post_id: 25, can_owe: true, can_reply: true, joined: false },
+  { user_id: 2, post_id: 26, can_owe: true, can_reply: true, joined: true, joined_at: "2012-10-03 22:36:00" },
+  { user_id: 3, post_id: 26, can_owe: true, can_reply: true, joined: false },
+  { user_id: 2, post_id: 27, can_owe: true, can_reply: true, joined: true, joined_at: "2012-10-08 15:31:00" },
+  { user_id: 3, post_id: 27, can_owe: true, can_reply: true, joined: false },
+  { user_id: 2, post_id: 28, can_owe: true, can_reply: true, joined: true, joined_at: "2012-10-09 12:27:00" },
+  { user_id: 3, post_id: 28, can_owe: true, can_reply: true, joined: false },
+  { user_id: 2, post_id: 29, can_owe: true, can_reply: true, joined: true, joined_at: "2012-10-10 21:38:00" },
+  { user_id: 3, post_id: 29, can_owe: true, can_reply: true, joined: false },
+  { user_id: 2, post_id: 30, can_owe: true, can_reply: true, joined: true, joined_at: "2012-10-11 17:19:00" },
+  { user_id: 3, post_id: 30, can_owe: true, can_reply: true, joined: false },
+  { user_id: 2, post_id: 31, can_owe: true, can_reply: true, joined: true, joined_at: "2012-10-12 16:02:00" },
+  { user_id: 3, post_id: 31, can_owe: true, can_reply: true, joined: false },
+  { user_id: 2, post_id: 32, can_owe: true, can_reply: true, joined: true, joined_at: "2015-07-22 17:44:00" },
+  { user_id: 3, post_id: 32, can_owe: true, can_reply: true, joined: true, joined_at: "2015-07-22 21:42:00" },
+])
+
+puts "Setting up post views..."
+PostView.create!([
+  { post_id: 1, user_id: 5, read_at: "2018-12-06 00:05:13" },
+  { post_id: 1, user_id: 1, read_at: "2018-12-06 00:05:13" },
+  { post_id: 1, user_id: 3, read_at: "2019-05-06 23:07:00" },
+  { post_id: 2, user_id: 3, read_at: "2019-06-22 07:40:11" },
+  { post_id: 4, user_id: 3 },
+  { post_id: 3, user_id: 3, read_at: "2019-06-06 01:53:29" },
+  { post_id: 32, user_id: 3, read_at: "2015-07-23 00:57:00" },
+])
+
+FlatPost.regenerate_all
```

Comment:
This depends heavily on background processing existing; not saying it's the wrong thing to do, but we might want it to be :bug: called out more obviously that this, unlike everything else, might be delayed in appearing in the SQL dataset.

Reply:
I can put a puts there about queueing Flat Plot generation?

Reply:
Does that work?